index.d.ts

```diff
@@ -68,12 +68,12 @@ export interface StaticPathSpec {
 }
 
 export interface StaticDocumentSpec {
-  document: string;
+  document: SwaggerSchema.Spec | object;
```

Comment:
Why also including `object` here?

Reply:
Because otherwise OpenAPI3 specs would not be allowed. I am happy to integrate

Reply:
So object is a catch all safety feedback in lack of knowing what specs have to be supported.
Just SwaggerSchema.Spec is not enough since it will only allow V2 Specs.
Hope i could make my point clear.

Reply:
Would rather us `Record<string, unknown>` instead of `object`. 

But also I don't see why we shouldn't use their official types package

Reply:
So, which option is prefered?
A) change object to Record<string, unknown>
or
B) import types from "openapi-types"

I was not aware of that these types were official. Knowing that i strongly tend to option B.

Reply:
I prefer `openapi-types` path too. This plung supports only version 2, though. Are we introducing incompatibilities/breaking changes? If so I would go with `Record<string, unknown>`  for the moment.